Type Conversions/Type Conversions/FromStr/task-info.yaml

```diff
@@ -3,12 +3,12 @@ files:
 - name: src/lib.rs
   visible: true
   placeholders:
-  - offset: 219
-    length: 665
+  - offset: 584
+    length: 872
     placeholder_text: /* TODO */
 - name: src/main.rs
   visible: true
 - name: Cargo.toml
   visible: true
 - name: tests/tests.rs
-  visible: false
+  visible: true
```

Comment:
I thought so because otherwise, we would probably need to describe the test cases in the task description. Also, Hint 3 suggests to check out the tests.

Reply:
Then sure, let's leave this way